src/System.Console/tests/System.Console.Tests.builds

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" >
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Console.Tests.csproj" >
+      <OSGroup>FreeBSD</OSGroup>
+    </Project>
+    <Project Include="System.Console.Tests.csproj" >
+      <OSGroup>Linux</OSGroup>
+    </Project>
+    <Project Include="System.Console.Tests.csproj" >
+      <OSGroup>OSX</OSGroup>
+    </Project>
+    <Project Include="System.Console.Tests.csproj" >
+      <OSGroup>Windows_NT</OSGroup>
+    </Project>
+  </ItemGroup>
```

Comment:
Currently, yes...however, they are all called out in the src builds file so the test builds file needs to have them as well